gcp/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "google" {}
```

Comment:
The customer can't use it as a module if you add the providers 

Better to add it to README with an example 

```
module "backed-nfs-server" {
  source = "github.com/env0/k8s-modules//gcp/pd-backed-nfs-server"
}
```

Reply:
The customer can use the individual modules if they want.
The full "gcp" folder should be possible to use as a "main stack", which is how we deployed this to our cluster.